packages/fast-animation/package.json

```diff
@@ -77,8 +77,8 @@
     "postcss": "^5.2.16",
     "postcss-loader": "^3.0.0",
     "prettier": "1.14.3",
-    "react": "^15.6.2",
-    "react-dom": "^15.6.2",
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0",
```

Comment:
Can you elaborate on choosing ^16.3 here? Is it essentially the lowest common denominator for our packages? I'm assuming there's intent with this specific version since we're jumping a major version.